packages/server/postgres/migrations/1685721573097_addPrePostMortemTemplates.ts

```diff
@@ -0,0 +1,1110 @@
+    .map(
+      (name, idx) =>
+        (idx === 0 ? name.charAt(0).toLowerCase() : name.charAt(0).toUpperCase()) + name.slice(1)
+    )
+    .join('')
+    .trim()
+  return `${cleanedName}${type === 'template' ? 'Template' : 'Prompt'}`
+}
+
+const makeTemplate = (template: Template): ReflectTemplate => ({
+  createdAt,
+  id: makeId(template.name, 'template'),
+  isActive: true,
+  name: template.name,
+  orgId: 'aGhostOrg',
+  scope: 'PUBLIC',
+  teamId: 'aGhostTeam',
+  type: 'retrospective',
+  updatedAt: createdAt,
+  isStarter: false,
+  isFree: true,
+  illustrationUrl:
+    template.type === 'postmortem' ? 'postMortemTemplate.png' : 'preMortemTemplate.png',
+  mainCategory: template.type,
+  lastUsedAt: null,
+  parentTemplateId: null
+})
+
+const promptColors = [
+  PALETTE.JADE_400,
+  PALETTE.TOMATO_500,
+  PALETTE.GOLD_300,
+  PALETTE.LILAC_500,
+  PALETTE.SKY_300,
+  PALETTE.TERRA_300,
+  PALETTE.FUSCIA_400,
+  PALETTE.SLATE_700
+]
+
+type PromptInfo = {
+  question: string
+  description?: string
+  templateId: string
+  sortOrder: number
+}
+
+const makePrompt = (promptInfo: PromptInfo, idx: number): RetrospectivePrompt => {
+  const {question, description, templateId, sortOrder} = promptInfo
+  const paletteIdx = idx > promptColors.length - 1 ? idx % promptColors.length : idx
+  const groupColor = promptColors[paletteIdx]
+  return {
+    createdAt,
+    description: description ?? '',
+    groupColor,
+    id: makeId(`${templateId}:${question}`, 'prompt'),
+    question,
+    sortOrder,
+    teamId: 'aGhostTeam',
+    templateId,
+    updatedAt: createdAt,
+    removedAt: null
+  }
+}
+
+const templates = NEW_TEMPLATE_CONFIGS.map((templateConfig) => makeTemplate(templateConfig))
+let colorIndex = 0
+const reflectPrompts = NEW_TEMPLATE_CONFIGS.map((templateConfig) => {
+  return templateConfig.prompts.map((prompt, idx) => {
+    colorIndex++
+    return makePrompt(
+      {
+        question: prompt.question,
+        description: prompt.description,
+        sortOrder: idx,
+        templateId: makeId(templateConfig.name, 'template')
+      },
+      colorIndex
+    )
+  })
+}).flat()
+
+const connectRethinkDB = async () => {
+  const {hostname: host, port, pathname} = new URL(process.env.RETHINKDB_URL!)
+  await r.connectPool({
+    host,
+    port: parseInt(port, 10),
+    db: pathname.split('/')[1]
+  })
+  return r as any as ParabolR
+}
+
+export async function up() {
+  const pgp = pgpInit()
+  const pg = pgp(getPgConfig())
+  const columnSet = new pgp.helpers.ColumnSet(
+    [
+      'id',
+      'createdAt',
+      'isActive',
+      'name',
+      'teamId',
+      'updatedAt',
+      'scope',
+      'orgId',
+      'type',
+      'illustrationUrl',
+      'mainCategory',
+      {name: 'isStarter', def: false},
+      {name: 'isFree', def: false}
+    ],
+    {table: 'MeetingTemplate'}
+  )
+  const insert = pgp.helpers.insert(templates, columnSet)
+  await pg.none(insert)
+  try {
+    await connectRethinkDB()
```

Comment:
-1 can remove RethinkDB, we've officially stopped writes to it for this table 🎉

Reply:
@mattkrick I'm confused. Did we migrate `ReflectPrompt`?